src/requests/typings/function.ts

```diff
@@ -45,6 +48,26 @@ export type GraphQLQueryResponse = {
   extensions?: Record<string, unknown>
 }
 
+type AppEventFilter<T> = {
+  type: typeof APP_EVENT_FILTER
+  entityProps: T
+  entityAction: string // 'create' | 'publish' etc etc.
+}
+// TODO: use generic to DRY this up?
```

Comment:
Can we get rid of this comment now?

Reply:
it looks like it! but also we need a TODO to make this exhaustive of all of the other entity types/actions that an app event subscription can receive (e.g. all of these that apply to AppEvents https://www.contentful.com/developers/docs/webhooks/content-events/)

eligible app event topics/actions
![image](https://github.com/contentful/node-apps-toolkit/assets/492573/6957606a-5972-4206-8e69-680bcf9bddf2)

Reply:
I removed this comment `// TODO: use generic to DRY this up?` and updated a comment to reflect that we need to update the app event subscription entities and topics/actions `// TODO: add all of the other app event subscription entities and topics/actions`